server/templates/base.html

```diff
@@ -1,33 +1,33 @@
 {#-
- Copyright 2020 Google LLC
+Copyright 2020 Google LLC
```

Comment:
curious why the formatting changes? was it from linting?

Reply:
Yes, these are caused by running ./run_test.sh -f